FAQ.rst

```diff
@@ -43,8 +43,13 @@ Why does the system rewrites a configuration file(s) that I have manually edited
 How can I modify an internal value of some service |omv| has control over?
 	Read :doc:`here <various/advancedsettings>` for advanced configurations.
 
+<<<<<<< HEAD
```

Comment:
Are these changes on purpose? Maybe you should update your repo, i had to rebase it some time ago to get rid of useless commits. I know, bad idea on a public repo, but for the moment there is less activity.

Reply:
I had a conflict when merging from upstream for this file. I though I solved the conflict but it wasn’t.